src/ServiceControl/Operations/Audit/ProcessAudits.cs

```diff
@@ -117,14 +132,35 @@ public void Stop()
                 }
 
                 meter.Mark(processedFiles.Count);
+            }
+            catch (Exception ex)
+            {
+                logger.Error("Something went wrong ingesting a batch", ex);
```

Comment:
True. Should each one trigger the breaker or should the breaker be triggered once if there were any failures?